coding_style/README.adoc

```diff
@@ -161,6 +166,17 @@ Rationale:: https://yaml.org/type/bool.html[YAML 1.1] allows all variants wherea
   your task will likely need options such as `changed_when:` and maybe `check_mode:`).
 * Anytime `command` or `shell` modules are used, add a comment in the code with justification to help with future maintenance.
 * Use the `| bool` filter when using bare variables (expressions consisting of just one variable reference without any operator) in `when`.
+* No single task file should contain more than 30 tasks.  Complex task files should be broken down into discrete parts.
+* The maximum block depth within a single task file should be 2
```

Comment:
No should, avoid concrete number, something "Limit the block depth within a single task file for readability reasons", and 2 can be part of the rationale (experience shows...) or the example.